### PR TITLE
Fix `docker-compose ps` output

### PR DESCRIPTION
**Pull request by joaothallis**
- fix expected State column in `docker-compose ps`
- fix `docker-compose ps` column name

### CODE REVIEW

File: content/tutorials/grafana-fundamentals.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ In this step, you'll set up the sample application, as well as supporting servic
    docker-compose ps
    ```
 
-   In the **State** column, it should say `Up` for all services.
+   In the **STATUS** column, it should say `running` for all services.
```
Comment (joaothallis) [inline content/tutorials/grafana-fundamentals.md:74]: https://docs.docker.com/engine/reference/commandline/compose_ps/#options

Comment (jdbaldry) [reply content/tutorials/grafana-fundamentals.md:74]: I just ran `docker-compose ps` and have the following columns:
```console
$ docker-compose ps
    Name                 Command             State    Ports
```

Is this change correct?

```console
$ docker-compose version

docker-compose version 1.29.2, build unknown
docker-py version: 5.0.3
CPython version: 3.9.10
OpenSSL version: OpenSSL 1.1.1n  15 Mar 2022
```

File: content/tutorials/grafana-fundamentals-cloud.md [hunk 2]
```diff
@@ -246,12 +246,12 @@ We have configured an email-based contact point to use a personal email. Now we
...
 1. Now scroll down to the **query B** box. For **Operation** choose `Classic condition`. [You can read more about classic and multi-dimensional conditions here](https://grafana.com/docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule/#single-and-multi-dimensional-rule). For conditions enter the following: `WHEN last() OF A IS ABOVE 0.2`
 1. In **Section 3**, enter `30s` for the **Evaluate every** field. For the purposes of this tutorial, the evaluation interval is intentionally short. This makes it easier to test. In the **For** field, enter `0m`. This setting makes Grafana wait until an alert has fired for a given time before Grafana sends the notification.
-1. In **Section 4**, you can add some sample text to your summary message. [Read more about message templating here](https://grafana.com/docs/grafana/latest/alerting/unified-alerting/message-templating/).
+1. In **Section 4**, you can add some sample text to your summary message. [Read more about message templating here](https://grafana.com/docs/grafana/next/alerting/contact-points/message-templating/).
```
Comment (gguillotte-grafana) [inline content/tutorials/grafana-fundamentals-cloud.md:254]: This doc is published to `/latest/`.

```suggestion
1. In **Section 4**, you can add some sample text to your summary message. For more information about message templating, refer to the [documentation](https://grafana.com/docs/grafana/latest/alerting/contact-points/message-templating/).
```

**Review by gguillotte-grafana — CHANGES_REQUESTED** [review]
Suggest using a link to `/latest/` docs instead of `/next/`.

**Comment by gguillotte-grafana** [comment]
@jdbaldry FYI, a CLA issue with you is blocking CI.

**Comment by jdbaldry** [comment]
Thanks for the heads up, just signed the CLA

**Comment by jdbaldry** [comment]
Hi @joaothallis, thanks again for contributing to the documentation.

Are you able to review the feedback above?

**Comment by jdbaldry** [comment]
Hi @joaothallis, I would like to thank you again for your PR.

We are refactoring tutorial content to live with the projects and products it relates to and as a result I am archiving this repository and its PRs.

This tutorial is being moved to the https://github.com/grafana/grafana repository so please do consider re-opening this PR there.

**Pull request closed without merging**